public/js/actions/AtomActions/getSuggestedContent.js

```diff
@@ -0,0 +1,74 @@
+import {getContentByTags} from '../../services/capi';
+import {fetchTargetsForAtomPath} from '../../services/TargetingApi';
+import {logError} from '../../util/logger';
+
+
+function requestSuggestedContent(atomType, id) {
+  return {
+    type:       'SUGGESTED_CONTENT_GET_REQUEST',
+    atomType:   atomType,
+    id:         id,
+    receivedAt: Date.now()
+  };
+}
+
+function receiveSuggestedContent(suggestedContent) {
+  return {
+    type:       'SUGGESTED_CONTENT_GET_RECEIVE',
+    suggestedContent: suggestedContent,
+    receivedAt: Date.now()
+  };
+}
+
+function errorReceivingSuggestedContent(error) {
+  logError(error);
+  return {
+    type:       'SHOW_ERROR',
+    message:    'Could not get atom usages',
+    error:      error,
+    receivedAt: Date.now()
+  };
+}
+
+/**
+ * Gets tags from Target api.
+ * Queries capi for content with those tags, filtering out
+ * any content which already have this atom.
+ */
+export function getSuggestedContent(atomId, atomType) {
+  const path = `/atom/${atomType.toLowerCase()}/${atomId}`;
+  const pluralType = `${atomType}s`;
+
+  return dispatch => {
+    dispatch(requestSuggestedContent());
+    return fetchTargetsForAtomPath(path)
+      .then((targets) => {
+        return targets.map((target) => {
+          return target.tagPaths;
+        });
+      })
+      .then(tags => {
```

Comment:
you can spare one level of nesting this way:
```
.then(tags => getContentByTags(tags, pluralType))
.then(content => {
    ...
})
.catch(error => {
    dispatch(errorReceivingSuggestedContent(error));
});
```

it won't change the world, I know 😅